monai/transforms/utility/array.py

```diff
@@ -202,43 +202,57 @@ def __call__(self, img: NdarrayOrTensor) -> NdarrayOrTensor:
 class EnsureChannelFirst(Transform):
     """
     Automatically adjust or add the channel dimension of input data to ensure `channel_first` shape.
-    It extracts the `original_channel_dim` info from provided meta_data dictionary.
-    Typical values of `original_channel_dim` can be: "no_channel", 0, -1.
-    Convert the data to `channel_first` based on the `original_channel_dim` information.
+
+    This extracts the `original_channel_dim` info from provided meta_data dictionary or MetaTensor input. This value
+    should state which dimension is the channel dimension so that it can be moved forward, or contain "no_channel" to
+    state no dimension is the channel and so a 1-size first dimension is to be added.
+
+    Args:
+        strict_check: whether to raise an error when the meta information is insufficient.
+        add_channel_default: If True and the input image `img` is not a MetaTensor and `meta_dict` is not given,
+            or `img` is a MetaTensor but doesn't specify channel dimension, use the value is `no_channel`
     """
 
     backend = [TransformBackends.TORCH, TransformBackends.NUMPY]
 
-    def __init__(self, strict_check: bool = True):
-        """
-        Args:
-            strict_check: whether to raise an error when the meta information is insufficient.
-        """
+    def __init__(self, strict_check: bool = True, add_channel_default=True):
         self.strict_check = strict_check
+        self.add_channel_default = add_channel_default
```

Comment:
I think the argument name `add_channel_default` may be a little confused, perhaps name such as  `force_add_channel` maybe more  easier to understand.

Reply:
I think it provides a default value only if channel_dim is None, so it's not forcing the value...

Reply:
It doesn't force adding the channel no, but if that was desired for some reason we should undeprecate `AddChannel`.